pkg/sql/opt/optbuilder/util.go

```diff
@@ -758,6 +761,9 @@ func tableOrdinals(tab cat.Table, k columnKinds) []int {
 	for i := 0; i < n; i++ {
 		col := tab.Column(i)
 		if shouldInclude[col.Kind()] {
+			if k.excludeVirtualComputedMutations && col.IsVirtualComputed() && col.IsMutation() {
```

Comment:
should we check if the mutation is in `DROP` direction?

Reply:
This mimics the behaviour that addComputedColsForTable, which generally skips all of these computed columns ignoring the direction